spec/spec_helper.rb

```diff
@@ -16,6 +16,12 @@
 # users commonly want.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+RSpec.configure do |config|
+  config.before(:suite) do
+    Rails.application.load_seed # loading seeds
```

Comment:
Mentioned yesterday but I think some tests might require the test db's to start off as empty, (not 100% sure on that though) so potentially this might need to be converted to a helper method which is called optionally at the start of tests, or for example this before each block is added in the spec files specifically for posts/comments